### PR TITLE
Enhancing redant_test_main

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Code changes:
1. Added file accessibilty check for config file.
2. Test list builder invocation added inside the main.

Updates: #103
Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: redant_test_main.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from test_list_builder import TestListBuilder
 
 
-def is_file_accessible(path: str, mode: str='r') -> bool:
+def is_file_accessible(path: str, mode: str = 'r') -> bool:
```
Comment (nishith-vihar) [inline redant_test_main.py:13]: There is an issue mentioned by Barak in redant MOMs regarding the is_file_accessible function. I think it should first be discussed in the meeting.

Comment (srijan-sivakumar) [reply redant_test_main.py:13]: The issue raised in MOM is about optimal manner but wrt pythonic way of checking if a file is accessible, the above function handles the case optimally. Do you have any point wit regards to the implementation of this function ?

Comment (nishith-vihar) [reply redant_test_main.py:13]: No. Nothing else

**Review by nishith-vihar — APPROVED** [review]

File: redant_test_main.py [hunk 2]
```diff
@@ -38,8 +63,15 @@ def main():
...
     args = pars_args()
 
-    if args.config_file:
+    if is_file_accessible(args.config_file):
         ParamsHandler.get_config_hashmap(args.config_file)
+    else:
+        print(f"The config file at {args.config_file} is not accessible.")
+        return -1
+
+    test_cases_list = TestListBuilder.create_test_list([args.test_dir])
```
Comment (nishith-vihar) [inline redant_test_main.py:72]: Is args.test_dir a single string consisting of multiple test dirs?

**Pull request merged**